src/altinn-app-frontend/src/components/summary/SummaryWrapper.test.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { screen, render } from '@testing-library/react';
+
+import SummaryWrapper from './SummaryWrapper';
+
+describe('SummaryWrapper', () => {
+const defaultProps = {
+  onChangeClick: () => { return },
+  changeText: 'some text on a button',
+  label: <h3>label text</h3>
+}
+  test('should render the wrapper with div inside', () => {
+    render(<SummaryWrapper {...defaultProps}>
+      <div data-testid={'innermost'}>
+        <p>Something</p>
+      </div>
+    </SummaryWrapper>)
+    expect(screen.queryByRole('button')).not.toBeNull();
+    expect(screen.getByRole('heading')).toBeInTheDocument();
+    expect(screen.getByRole('button')).toBeInTheDocument();
```

Comment:
Isnt
```
expect(screen.queryByRole('button')).not.toBeNull();
```
and 
```
expect(screen.getByRole('button')).toBeInTheDocument();
```
the same thing?

Reply:
Indeed they are checking for the same, I was thinking more in the lines of the other tests that use the queryByRole to verify the opposite. I guess line 20 would fail if 18 wasn't implicitly passed so I can take it away.